datasette_write_ui/table.ts

```diff
@@ -71,13 +71,10 @@ class RowIcon {
     this.toggle = this.toggle.bind(this);
     this.addButton = this.addButton.bind(this);
 
-    this.root = html`<span class="row-icon">
-      <button class="icon">&#9881;</button>
-      <div class="menu"></div>
-    </span>`;
+    this.root = html`<span class="row-icon"><button class="icon">&#9881;</button><div class="datasette-write-ui-menu"></div></span>`;
```

Comment:
Removing the whitespace from this fixed a separate bug: the big chunk of empty space at the bottom of the table, which results in an occasional scroll bar:

<img width="648" alt="CleanShot 2023-09-09 at 15 05 14@2x" src="https://github.com/datasette/datasette-write-ui/assets/9599/8c5333d2-1426-40d5-989a-117d6eac101e">

This is because of the `white-space: pre-wrap` that's in play here, to avoid long IDs wrapping.